api-gateway/kong/examples/server-api-example/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:12.18-alpine
+
+WORKDIR /usr/src/app
+
+COPY server-api-example/package.json package.json
+COPY server-api-example/package-lock.json package-lock.json
+COPY server-api-example/server.js server.js
```

Comment:
It'd be better to install the dependencies before copying the source code to avoid reinstall them if the source code changes.

Reply:
I wasn't very careful because it was just an example, but I can change